src/modules/blocks/event-datetime/style.pcss

```diff
@@ -46,6 +47,7 @@
 
 	.tribe-editor__btn--label {
 		text-align: left;
+		flex: 1 1 100%;
```

Comment:
If you're using the `flex` approach, every flex child is going to need a `flex` property, so here would be the price and timezone (I believe). Different browsers have different defaults for the `flex` property and will not behave the same way.